lib/Elastica/Search.php

```diff
@@ -476,12 +476,11 @@ public function count($query = '', $fullResult = false)
 
         $query = $this->getQuery();
         $path = $this->getPath();
-
         $response = $this->getClient()->request(
             $path,
             Request::GET,
             $query->toArray(),
-            array(self::OPTION_SEARCH_TYPE => self::OPTION_SEARCH_TYPE_COUNT)
+            array(self::OPTION_SEARCH_TYPE => self::OPTION_SEARCH_TYPE_QUERY_THEN_FETCH)
```

Comment:
Yes, but Elasticsearch documentation says that `search type` of `count` is deprecated?  And that `count` doesn't have any advantage over `query then fetch` (provided size of query is 0)

https://www.elastic.co/guide/en/elasticsearch/reference/2.x/search-request-search-type.html